definitions/divorce/json/CaseEventToFields/CaseEventToFields-deemed-and-dispensed-nonprod.json

```diff
@@ -50,6 +51,21 @@
     "PageDisplayOrder": 2,
     "PageColumnNumber": 1,
     "PageShowCondition": "ServiceApplicationGranted=\"No\"",
+    "ShowSummaryChangeOption": "Yes",
+    "ShowSummaryContentOption": "Yes"
+  },
+  {
+    "LiveFrom": "03/08/2020",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "makeServiceDecision",
+    "CaseFieldID": "ServiceRefusalDraft",
+    "PageFieldDisplayOrder": 1,
+    "DisplayContext": "READONLY",
+    "PageID": "makeServiceDecision3",
+    "PageLabel": "Reason for refusal - Make Service Decision",
+    "PageDisplayOrder": 3,
+    "PageColumnNumber": 1,
+    "PageShowCondition": "ServiceApplicationRefusalReason=\"*\"",
```

Comment:
Adding 3 step for makeServiceDecision flow to display the generated draft the download document

Reply:
Do you really need this condition? What's the condition here?

Reply:
To only show when the user has added some refusal reason in the page before. This page shows the link to the draft refusal document and if no refusal reason then the draft cant be generated.